book/layout.md

```diff
@@ -310,9 +297,8 @@ complex layout trees!
 ![An example of an HTML tree and the corresponding layout tree](im/layout-tree.png)
 :::
 
-Oh, and you might also notice that the text on these web pages is now
```

Comment:
Is this obsolete?

Reply:
So, in the old version of the chapter, we add the layout tree and initially implement a "stop-gap" painting strategy (wherein each node paints itself during layout, and propagates display lists up three tree). Then a little later, we add a proper, separate paint phase.

In the proposed new version, we just never implement the stopgap. The benefit is we don't have to describe this straw-man stop-gap and also you don't have to remember to take it out. The downside is that the browser now doesn't show anything until we implement paint properly—so this observation about all the text overlapping doesn't make sense anymore. Hence it is removed. It's possible to maybe get the benefits of both, by moving painting up and then rejiggering the whole chapter, but I wasn't really confident in attempting that this late stage in editing.